ir-engine/ir-engine
# IR-3575 playmode on scene change

## Summary
moving start/stop playmode into its own functions file, try exiting play mode on scene loading

## References
https://tsu.atlassian.net/browse/IR-3575

## QA Steps
open a scene, enter playmode, load a new scene without manually leaving playmode.

you should be able to properly see playmode has ended once the new scene is loaded, and you can enter/exit playmode as expected in the new scene

Repeat to your hearts content
